### ISSUE TITLE
a11y: more stories not loading when I use keyboard to go to the last story in the dashboard

### ISSUE THREAD

**Issue by o-fernandez** [GoogleForCreators/web-stories-wp#4723]
## Bug Description

Pagination/new stories don't load/get triggered when I'm using the keyboard to view stories. 

## Expected Behaviour

When I use keyboard arrow keys to get to the last row of stories, more stories should be loaded. 

## Screenshots

1. Open My Stories
2. Focus on story grid
3. Use keyboards to navigate to the end of the list

Observe: no new stories loaded even if there are more stories to load, if you use the mouse to scroll down you'll see them load. 

## Additional Context

- Plugin Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by swissspidy** [comment]
The loading of new stories should also get announced to screen readers.

But triggering the loading when using the keyboard to go through the list might be confusing. A hidden button to load more stories (that is only visible when navigating with the keyboard) might be more user friendly

**Comment by csossi** [comment]
Verified in QA
